flash_attn/flash_attention.py

```diff
@@ -103,6 +103,15 @@ def __init__(self, embed_dim, num_heads, bias=True, batch_first=True, attention_
         self.inner_attn = FlashAttention(attention_dropout=attention_dropout, **factory_kwargs)
         self.out_proj = nn.Linear(embed_dim, embed_dim, bias=bias, **factory_kwargs)
 
+        if 'qkv_weight' in init_para.keys():
+            self.Wqkv.weight = torch.nn.Parameter(init_para["qkv_weight"])
+        if 'qkv_bias' in init_para.keys():
+            self.Wqkv.bias = torch.nn.Parameter(init_para["qkv_bias"])
+        if 'proj_weight' in init_para.keys():
+            self.out_proj.weight = torch.nn.Parameter(init_para["proj_weight"])
+        if 'proj_bias' in init_para.keys():
+            self.out_proj.bias = torch.nn.Parameter(init_para["proj_bias"])
+
```

Comment:
wouldn't [loading a state_dict](https://pytorch.org/tutorials/beginner/saving_loading_models.html#save-load-state-dict-recommended) be a better solution?